website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -170,17 +170,22 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-ide/python-black.gif" width="95%" title="Format Python files using Black."/>
 
-## Troubleshooting
+## FAQs
 
 <details>
 <summary>Can I nest <code>.sqlfluff</code> files?</summary>
 
-To ensure optimal code quality, consistent code, and styles, it's highly recommended you have one main `.sqlfluff` configuration file in the root folder of your project. Having multiple files can result in various different SQL styles in your project. <br /><br />
+To ensure optimal code quality, consistent code, and styles &mdash; it's highly recommended you have one main `.sqlfluff` configuration file in the root folder of your project. Having multiple files can result in various different SQL styles in your project. <br /><br />
 
 However, you can customize and include an additional child `.sqlfluff` configuration file within specific subfolders of your dbt project. <br /><br />By nesting a `.sqlfluff` file in a subfolder, SQLFluff will apply the rules defined in that subfolder's configuration file to any files located within it. The rules specified in the parent `.sqlfluff` file will be used for all other files and folders outside of the subfolder. This hierarchical approach allows for tailored linting rules while maintaining consistency throughout your project. Refer to [SQLFluff documentation](https://docs.sqlfluff.com/en/stable/configuration.html#configuration-files) for more info.
 
 </details>
 
+<details>
+<summary>Can I run SQLFluff commands form the terminal?</summary>
+
+Running SQLFluff commands from the terminal isn't supported, but it's currently on the Product team's roadmap for future support.
```

Comment:
i've been taught to avoid adding this kind of info in the user docs for these reasons:
- it can be seen as a promise we're making and could be a liability for us (especially if the roadmap changes)
- it can create support churn when people reach out asking when the functionality will be available.

```suggestion
Currently, running SQLFluff commands from the terminal isn't supported. 
```